rskj-core/src/main/java/co/rsk/peg/lockingcap/constants/LockingCapTestNetConstants.java

```diff
@@ -0,0 +1,34 @@
+package co.rsk.peg.lockingcap.constants;
+
+import co.rsk.bitcoinj.core.Coin;
+import co.rsk.peg.vote.AddressBasedAuthorizer;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.bouncycastle.util.encoders.Hex;
+import org.ethereum.crypto.ECKey;
+
+public class LockingCapTestNetConstants extends LockingCapConstants {
+
+    private static final LockingCapTestNetConstants instance = new LockingCapTestNetConstants();
+
+    private LockingCapTestNetConstants() {
+        List<ECKey> increaseAuthorizedKeys = Arrays.stream(new String[]{
```

Comment:
nit: for this scenarios, we can also use `Arrays.asList(...)`

Reply:
Thanks for your feedback, @apancorb! For this opportunity, we can't apply `Arrays.asList(...)` to replace `new String[]` since the `Arrays.stream` method receives an `array` and not a `List<>`.

Reply:
i think we can use `Collections.unmodifiableList(Stream.of(...` like in FederationConstants. Check [FederationMainNetConstants](https://github.com/rsksmart/rskj/blob/federation-support-refactor-integration/rskj-core/src/main/java/co/rsk/peg/federation/constants/FederationMainNetConstants.java#L22)

Reply:
Done!

Reply:
You should be able to do so as something like this:

```java
List<ECKey> increaseAuthorizedKeys = Arrays.asList(
    "0470...",
    "045e...",
    "0455...")
   .stream()
   .map(hex -> ECKey.fromPublicOnly(Hex.decode(hex)))
   .collect(Collectors.toList());
```

But I agree with what Julia proposed this way we are more consistent in the codebase.